### DISCUSSION TITLE
Failed to local RAG and Webpage load after Upgrade to 0.3.4

### DISCUSSION THREAD

**Discussion by AeneasZhu** [open-webui/open-webui#3142] [General]
# Bug Report

## Description

**Bug Summary:**
After I upgrade open-webui to 0.3.4, it fails to load local documents and also cannot load any webpage anymore.

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Open WebUI Version:** 0.3.4
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** Win11
- **Browser (if applicable):** Brave latest

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots
![image](https://github.com/open-webui/open-webui/assets/44701446/ef0c5c01-15c9-4f79-8763-3b72a771c09a)

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
![image](https://github.com/open-webui/open-webui/assets/44701446/ef53f485-c701-4e4f-846d-0099f09efb20)

## Installation Method

pip install open-webui

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Unable to reproduce the issue here, I'd suggest you try using other documents and see if you experience the same issue. Also try resetting vector DB as well.

**Comment by AeneasZhu** [comment]
> Unable to reproduce the issue here, I'd suggest you try using other documents and see if you experience the same issue. Also try resetting vector DB as well.

![image](https://github.com/open-webui/open-webui/assets/44701446/6a925f6e-5ada-409e-8dce-7e0ae1863290)

I have tried all you told. It didn't work.

![image](https://github.com/open-webui/open-webui/assets/44701446/70f2d5ca-0305-427a-a864-3a38b8bab32e)

**Reply by tjbck** [reply]
![image](https://github.com/open-webui/open-webui/assets/25473318/c9247a8e-d5c0-48b7-aae1-4367318d136b)

![image](https://github.com/open-webui/open-webui/assets/25473318/9242c7c1-815e-4965-8cc6-52bda9f0ed09)

I've tried everything you've tried, it worked. If you figure out what the culprit was, please share with us!

**Reply by AeneasZhu** [reply]
> ![image](https://private-user-images.githubusercontent.com/25473318/339585468-c9247a8e-d5c0-48b7-aae1-4367318d136b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzNzEyODgsIm5iZiI6MTcxODM3MDk4OCwicGF0aCI6Ii8yNTQ3MzMxOC8zMzk1ODU0NjgtYzkyNDdhOGUtZDVjMC00OGI3LWFhZTEtNDM2NzMxOGQxMzZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE0VDEzMTYyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTllNGVkOTdiZTlmYmUxMDI5YzQyMGVkMmFlZTdlNDhlM2RiOGU3NGY5ZTIwNjkxZDIyMGRlODAxZWU0NGYxZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JKffXchEa3Z2YfEZigSS8ml3XVb4k_OkuOz3H9CIvz8)
> 
> ![image](https://private-user-images.githubusercontent.com/25473318/339585611-9242c7c1-815e-4965-8cc6-52bda9f0ed09.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzNzEyODgsIm5iZiI6MTcxODM3MDk4OCwicGF0aCI6Ii8yNTQ3MzMxOC8zMzk1ODU2MTEtOTI0MmM3YzEtODE1ZS00OTY1LThjYzYtNTJiZGE5ZjBlZDA5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE0VDEzMTYyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNmNmJhMTc0MGI5ODAyYTZmYzM3MmExNWU2NTUyMzBjN2Q0MTI4ZjNiNzE1ZDc3OTY0YmU0OTUwNmViNjlkYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qoXeK0M8sES5P1cUYPeF1Mhw0q6-B_d_PWpWx0sqmuQ)
> 
> I've tried everything you've tried, it worked. If you figure out what the culprit was, please share with us!

It seems that it is the `pip install open-webui --upgrade` that causes this issue. After I remove the upgraded version and reinstall the latest one in a new pyvenv, the problem is fixed.  Could there be any improvements for upgrade via pip? (Since the country I live restrict the access to docker or github, and the mirror of docker in my country is broken, I have to install it via pip). 

Anyway, thanks for your help and I appreciate your work. Open-Webui helps me a lot in my job and study.
